src/main/app/controller/GetCaseController.ts

```diff
@@ -75,6 +75,7 @@ export class GetCaseController {
 
   public async createC100ApplicantCase(req: AppRequest, res: Response): Promise<void> {
     const userDeatils = req?.session?.user;
+
```

Comment:
Do not leave this whitespaces in PR. Its a small PR, easy to traverse through. when its a lengthy one, this files increases the count.